cmd/cagent/main.go

```diff
@@ -97,7 +98,7 @@ func main() {
 
 	cfg, err := cagent.HandleAllConfigSetup(*cfgPathPtr)
 	if err != nil {
-		log.Fatalf("Failed to handle cagent configuration: %s", err)
+		log.Fatal("Failed to handle cagent configuration: ", err.Error())
```

Comment:
Why change from `Fatalf` with proper formatting to `Fatal`?

Reply:
Because in case it just needs to append string no need to use a placeholder. It produces the same result

Reply:
We already had quite some issues because we took some shortcuts. IMHO `It produces the same result` is another argument for this. Let's go with explicit formatting.

Reply:
ok. I reverted this change